src/Integrations/Mailer/SettingsMailer.php

```diff
@@ -214,157 +226,181 @@ public function getSettingsData(string $formId): array
 				'tabsContent' => [
 					[
 						'component' => 'tab',
-						'tabLabel' => \__('Sender', 'eightshift-forms'),
+						'tabLabel' => \__('Settings', 'eightshift-forms'),
 						'tabContent' => [
 							[
-								'component' => 'input',
-								'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_NAME_KEY),
-								'inputFieldLabel' => \__('Name', 'eightshift-forms'),
-								'inputFieldHelp' => \__('Most e-mail clients show this instead of the e-mail address in the list of e-mails', 'eightshift-forms'),
-								'inputType' => 'text',
-								'inputIsRequired' => true,
-								'inputValue' => $this->getSettingsValue(self::SETTINGS_MAILER_SENDER_NAME_KEY, $formId),
-							],
-							[
-								'component' => 'divider',
-								'dividerExtraVSpacing' => 'true',
-							],
-							[
-								'component' => 'input',
-								'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_EMAIL_KEY),
-								'inputFieldLabel' => \__('E-mail', 'eightshift-forms'),
-								'inputFieldHelp' => \__('Shown in the <code>From:</code> field', 'eightshift-forms'),
-								'inputType' => 'text',
-								'inputIsEmail' => true,
-								'inputIsRequired' => true,
-								'inputValue' => $this->getSettingsValue(self::SETTINGS_MAILER_SENDER_EMAIL_KEY, $formId),
+								'component' => 'select',
+								'selectSingleSubmit' => true,
+								'selectName' => $this->getSettingsName(self::SETTINGS_MAILER_EMAIL_FIELD_KEY),
+								'selectFieldHelp' => \__('You must select what field is used as an email.', 'eightshift-forms'),
```

Comment:
```suggestion
								'selectFieldHelp' => \__('Required.', 'eightshift-forms'),
```
😄

Reply:
I would like to leave it as is because this better describes what this field is